pages/docs/guides/validate.md

```diff
@@ -0,0 +1,129 @@
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
+```
+
+# Validate AsyncAPI documents
+Validating an AsyncAPI document can mean one of two things: 
+1. Validation against the specification.
+2. Validation against the best practices or company governance rules.
+
+## Validate against specification
+Let's discuss options for validating against the specification. 
+
+This ensures that every content of the document is written in accordance with the AsyncAPI specification.
+
+There are several tool options you may select for validating against the specification: _AsyncAPI Studio_, _AsyncAPI CLI_, and _Parsers_.
+
+### AsyncAPI Studio validation 
+[AsyncAPI Studio](https://github.com/asyncapi/studio#readme) provides a visual and easy way to validate your AsyncAPI documents against the specification. 
+
+It uses the [AsyncAPI JavaScript parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and validate documents.
+
+Errors in your document are highlighted with a red underline, showing which lines are invalid. The console also provides feedback, allowing you to further troubleshoot with detailed error messages.
+
+When a document is invalid, it provides the following error: `Empty or invalid document please fix errors / define AsyncAPI document`.
+
+### AsyncAPI CLI validation 
+The following [AsyncAPI CLI](https://github.com/asyncapi/cli#installation) command validates AsyncAPI documents in your local computer or in automation:
+
+ ```
+ asyncapi validate asyncapi.yaml
+
+ ```
+
+> You can also open the AsyncAPI Studio from the CLI by running the command `asyncapi start studio`
+
+### Parsers (code) validation 
+AsyncAPI provides official [JavaScript](https://github.com/asyncapi/parser-js) and [Go](https://github.com/asyncapi/parser-go) parsers for validating the JSON schema in your AsyncAPI documents. 
+
+<Remember>
+If you do not run your schemas through the parsers, the JSON schemas alone cannot complete the entire validation. 
+
+Learn more about [custom JSON Schemas validation needs](https://github.com/asyncapi/spec-json-schemas#custom-validation-needs).
+</Remember>
+
+## Validation against best practices or company governance rules
+Now let's discuss options for validating against best practices or company governance rules also known as **linting**.
+
+When AsyncAPI is used by various teams, you want to make sure they follow the same rules and are consistent across the organization. It is not enough to validate AsyncAPI documents against official specification rules. 
+
+An example would be that officially, `summary` property is optional, but in your organization, you want to make sure that it is always provided. You need a solution that enables you to enforce internal rules on AsyncAPI documents' providers.
```

Comment:
```suggestion
<Remember>
Let's discuss an example. While the `summary` property is optional in an AsyncAPI document, you could choose to require it for your organization. You would then implement a solution that enables you to enforce internal rules on AsyncAPI documents' providers.
```

Reply:
Hey @alequetzalli, what is the best way to format the `summary` property in the remember block? should it be in bold, italics or just as a normal word?
I tried using the `code` tag but didn't format it as I expected.